src/utils/dataFormatters/dataFormatters.ts

```diff
@@ -40,19 +40,33 @@ export const stringifyVdiskId = (id?: TVDiskID | TVSlotId) => {
     return id ? Object.values(id).join('-') : '';
 };
 
-export const formatUptimeInSeconds = (seconds: number) => {
-    const days = Math.floor(seconds / DAY_IN_SECONDS);
-    const remain = seconds % DAY_IN_SECONDS;
+/**
+ * It works well only with positive values,
+ * if you want to get negative formatted uptime, use some wrapper like getDowntimeFromDateFormatted
+ */
+export function formatUptimeInSeconds(seconds: number) {
```

Comment:
lets handle somehow negative values. As for me this function should work properly with all values. And we should use it for negative and positive uptime.